app/src/main/res/layout/activity_login.xml

```diff
@@ -5,15 +5,17 @@
     android:layout_width="match_parent"
     android:layout_height="match_parent"
     tools:context=".LoginActivity">
-
+    <!--    Todo: remove explicit set textColor black-->
```

Comment:
same comment as above, how can we do this and is it necessary?

Reply:
Created a theme in style.xml and set it to Login and CreateAccount Activity in the android manifest file.  It would be easier to change color in one place than changing each editText.